README.md

```diff
@@ -23,3 +23,16 @@ For more information about the OpenGitOps project, see: <https://github.com/open
 ## Code of Conduct
 
 The OpenGitOps community follows the [CNCF Code of Conduct](https://github.com/open-gitops/.github/blob/main/CODE_OF_CONDUCT.md).
+
+## Reference this Content
+
+A [BibTex](https://www.bibtex.org) code snippet to use as reference for the content available in this repository is listed below, make sure to change your site visit date accordinly.
+
+```
+@misc{open_gitops,
+  title        = {{GitOps Principles and Glossary}},
+  howpublished = {\url{https://github.com/open-gitops/documents}},
+  note         = {Accessed: YYYY-MM-DD. For an up-to-date list of authors, see \url{https://github.com/open-gitops/documents/blob/main/authors.md}. Release version 1.1.0},
```

Comment:
I would just change this to a direct link to the forthcoming release, since we plan to cut the release immediately after merging this PR.

```suggestion
  note         = {Accessed: YYYY-MM-DD. For an up-to-date list of authors, see \url{https://github.com/open-gitops/documents/blob/v1.1.0/authors.md}. Release version 1.1.0},
```

Reply:
good catch!

Reply:
fixed with https://github.com/open-gitops/documents/pull/78/commits/5b7f116a8c00ea940f3cb6663f06608ab9ea3cf2

Reply:
I know I suggested referencing the upcoming release, but since then we've discussed that it would be better to get the translations updated before cutting the v1.1.0 release.

So if we want merge this PR sooner than all translations are complete – which may take a while – we'll want to change the version back to what you had before:
- Release version "1.0.0" (instead of 1.1.0)
- link "blob/v1.0.0/authors.md" (instead of blog/v1.1.0)
- and year = "{2021}" (instead of {2024})

Reply:
this was solved by https://github.com/open-gitops/documents/commit/f7609a0fa3ca5d0e9ec4fe05366e4d0f0368a0c9